client-ui/src/app/components/lineage-overview/lineage-overview-details/lineage-overview-details.component.ts

```diff
@@ -27,25 +30,44 @@ import {AttributeVM} from 'src/app/model/viewModels/attributeVM';
 })
 export class LineageOverviewDetailsComponent {
 
-  @Input()
-  public embeddedMode: boolean
+  @Input() embeddedMode: boolean
 
-  constructor(private store: Store<AppState>) {
-  }
+  readonly selectedNodeUrl$: Observable<string | null>
 
-  public getLineageOverviewInfo = (): Observable<{ [key: string]: {} }> => {
-    return this.store.select('lineageOverview', "lineageInfo")
-  }
+  readonly operationDetails$: Observable<OperationDetailsVM>
 
-  public getDetailsInfo = (): Observable<OperationDetailsVM> => {
-    return this.store.select('detailsInfos')
-  }
+  readonly executedLogicalPlan$: Observable<ExecutedLogicalPlanVM>
 
-  public getOutputSchema = (operationDetails: OperationDetailsVM): AttributeVM[] => {
-    return operationDetails.schemas[operationDetails.output]
-  }
+  readonly lineageInfo$: Observable<{
+    targetNodeName: string;
```

Comment:
Is this semicolon necessary?

Reply:
I will be doing that all the time without ...